test/models/WebsitePaymentsModel/test.js

```diff
@@ -0,0 +1,141 @@
+      host = faker.random.string();
+      condition = { host };
+      sort = { createAt: 1 };
+
+      const websitePayment1 = {
+        host,
+        type: PAYMENT_TYPES.WRITE_OFF,
+        amount: 10,
+      };
+      const websitePayment2 = {
+        host,
+        type: PAYMENT_TYPES.REFUND,
+        amount: 15,
+      };
+      const websitePayment3 = {
+        host,
+        type: PAYMENT_TYPES.WRITE_OFF,
+        amount: 20,
+      };
+      const websitePayment4 = {
+        type: PAYMENT_TYPES.REFUND,
+        amount: 13,
+      };
+      websitePayments = [websitePayment1, websitePayment2, websitePayment3, websitePayment4];
+      for (const websitePayment of websitePayments) {
+        await WebsitePaymentsFactory.Create({ ...websitePayment });
+      }
+    });
+
+    it('Should return array 3 elements long', async () => {
+      const { result } = await WebsitePaymentsModel.find({ condition, sort });
+      expect(result).to.have.length(3);
+    });
+    it('Should return array without one element and when searching not to find it', async () => {
+      const { result } = await WebsitePaymentsModel.find({ condition, sort });
+      expect(result).to.not.include(websitePayments[3]);
+    });
+    it('Should return array whose first element will contain the smallest amount & the last largest', async () => {
+      const { result } = await WebsitePaymentsModel.find({ condition, sort });
+      expect(result[0].amount).to.be.eq(websitePayments[0].amount);
+      expect(result[2].amount).to.be.eq(websitePayments[2].amount);
+    });
+    it('Should check that the result will be undefined', async () => {
+      const { error } = await WebsitePaymentsModel.find(faker.random.string());
+      expect(error).to.be.undefined;
+    });
+  });
+
+  describe('On aggregate', () => {
+    let websitePayments, condition, name;
+    beforeEach(async () => {
+      await dropDatabase();
+      name = faker.name.firstName();
+      condition = [{ $match: { userName: name } }];
+      const websitePayment1 = { name };
+      const websitePayment2 = { name };
+      const websitePayment3 = { name: faker.name.firstName() };
+      const websitePayment4 = { name: faker.name.firstName() };
+      websitePayments = [websitePayment1, websitePayment2, websitePayment3, websitePayment4];
+      for (const websitePayment of websitePayments) {
+        await WebsitePaymentsFactory.Create({ ...websitePayment });
+      }
+    });
+    it('Should return an array in which the names match the condition', async () => {
+      const { result } = await WebsitePaymentsModel.aggregate(condition);
+      expect(result).to.have.length(2);
+    });
+    it('Should return an array that does not contain elements that satisfy the condition', async () => {
+      const { result } = await WebsitePaymentsModel.aggregate(condition);
+      expect(result).to.not.include(websitePayments[2]);
+      expect(result).to.not.include(websitePayments[3]);
+    });
+  });
```

Comment:
1)better be check that every result that you got has proper userName
2) 2 expect